sdcm/remote.py

```diff
@@ -243,7 +254,27 @@ def _ssh_ping(self, timeout=30, verbose=False):
             self.log.debug(details)
             return False
 
-    def receive_files(self, src, dst, delete_dst=False,  # pylint: disable=too-many-arguments,too-many-branches
+    def ssh_ping_thread(self):
+        while not self._ssh_up_thread_termination.isSet():
```

Comment:
use .wait(timeout) instead of is set, and make the timeout more then 5sec (the default timeout of the ssh_ping is 30sec)

Reply:
Fixed.